itchiodl/game.py

```diff
@@ -42,13 +42,14 @@ def __init__(self, data):
         else:
             self.publisher_slug = matches.group(1)
 
+        self.destination_path = Path(f"{self.publisher_slug}/{self.game_slug}")
```

Comment:
I don't see why this was changed to an fstring with no escaping. Surely the code on L48-52 is safer with the human readable names option

Reply:
Reverted it back